### DISCUSSION TITLE
Sharing website Domain (in Telegram) shows AstroWindTheme Image. Already changed screenshot and default image?!

### DISCUSSION THREAD

**Discussion by gangajamuna** [arthelokyo/astrowind#84] [Q&A]
Hello, 
when I share the site (Domain Link) that I build with the AstroWind Theme in Telegram the astrowind default Image is shown together with the Link.

I already updated the screenshot picture and default image. In the root folder and the assets.
after running the build the AstroWind Image is still showing up?!

Where is this coming from? Am I missing something here?

Thank you very much!

**Comment by prototypa** [comment]
Hello @gangajamuna 

I've been reviewing the code and doing some tests and so far everything looks good.

It seems to me that they may be due to the fact that social networks and other applications usually have a cache associated with the capture of url metadata. It might still keep the old image in cache for some time.

To check that when changing the image everything is ok, try the url on this site: https://metatags.io/

**Reply by gangajamuna** [reply]
Thanks for the Tip, didn't know metatags.io ! I saw that the og:image meta tag has the Image I want so I guess it is the cache of Telegram!
Thanks for your help! Much appreciated.
